src/openms_gui/include/OpenMS/VISUAL/sources.cmake

```diff
@@ -50,11 +50,11 @@ TOPPASToolVertex.h
 TOPPASTreeView.h
 TOPPASVertex.h
 TOPPASWidget.h
-TOPPViewIdentificationViewBehavior.h
 TOPPViewMenu.h
 #TOPPViewPreferences.h
```

Comment:
can that be removed?

Reply:
yes was renamed to TVIdentificationViewController